### PR TITLE
Add Portuguese language support

### PR DESCRIPTION
**Pull request by pablothedude**
This will merge the Portuguese translations into a new feature branch
with all recent changes. This is mostly concerning the changed metadata
endpoints and SFO callout logic.

This PR will only add PT as language. Some tasks still need to be done
but will be addressed later on in separate PR's.

- [ ] Fix checks with only allowed languages (helper)
- [ ] Check and validate missing PT translations
- [ ] Update migrations and test and validate with metadata push
- [ ] Move translations from parameters.yml to translations

### CODE REVIEW

File: src/OpenConext/EngineBlock/Metadata/Factory/Decorator/ProxiedIdentityProvider.php [hunk 1]
```diff
@@ -98,35 +98,4 @@ public function getSingleSignOnServices(): array
...
-     * Buisiness rule:
-     * displayname:en, name:en
-     */
-    public function getDisplayNameEn(): string
-    {
-        if (empty($this->entity->getDisplayNameEn())) {
-            return $this->entity->getNameEn();
-        }
-        return $this->entity->getDisplayNameEn();
-    }
```
Comment (MKodde) [inline src/OpenConext/EngineBlock/Metadata/Factory/Decorator/ProxiedIdentityProvider.php:131]: The removal of these methods seem unrelated to the PT language addition? Please explain.

**Comment by MKodde** [comment]
Are you planning on addressing the open todo's on this PR or on successive PR(s)?

**Comment by pablothedude** [comment]
I will address this later on in separate PR's.

File: src/OpenConext/EngineBlock/Metadata/Factory/Decorator/ProxiedIdentityProvider.php (hunk 1 above)
Comment (pablothedude) [reply src/OpenConext/EngineBlock/Metadata/Factory/Decorator/ProxiedIdentityProvider.php:131]: This logic was moved to the AbstractIdentityProvider as it seemed default behaviour to present a displayname. Also the displayname seems to be unrelated to proxying so therefore I removed it from this decorator.

Comment (MKodde) [reply src/OpenConext/EngineBlock/Metadata/Factory/Decorator/ProxiedIdentityProvider.php:131]: Did you check with the product manager if this behavioural change is expected behavior? It was added specifically for the IdP metadata which sometimes contained empty titles. Moving it to the abstract for all other IdP to benefit from this is probably fine. But might not?

**Comment by MKodde** [comment]
All systems go, except for this question: https://github.com/OpenConext/OpenConext-engineblock/pull/841#discussion_r405556312

File: src/OpenConext/EngineBlock/Metadata/Factory/Decorator/ProxiedIdentityProvider.php (hunk 1 above)
Comment (pablothedude) [reply src/OpenConext/EngineBlock/Metadata/Factory/Decorator/ProxiedIdentityProvider.php:131]: Validated the expected behaviour with @thijskh and this is always expected behaviour.

Comment (MKodde) [reply src/OpenConext/EngineBlock/Metadata/Factory/Decorator/ProxiedIdentityProvider.php:131]: In that case it's fine with me!
